common/scala/src/main/scala/whisk/core/database/CouchDbRestClient.scala

```diff
@@ -100,12 +100,12 @@ class CouchDbRestClient protected (system: ActorSystem, protocol: String, host:
         }
 
         futureBody.map { body =>
-            HttpRequest(method = method, uri = uri, headers = headers, entity = body)
+            HttpRequest(method=method, uri=uri, headers=headers, entity=body)
```

Comment:
I prefer the old version better (with spaces left/right of `=`.
Are you applying all this formatting automatically - if so, will there be an updated eclipse file?